lib/config/mysql_config.go

```diff
@@ -44,14 +54,21 @@ func (c *mysqlConfig) InitializeConfiguration() error {
 	}
 
 	// Create the database if it doesn't exist
-	if _, err := c.db.Exec("CREATE SCHEMA IF NOT EXISTS ? DEFAULT CHARACTER SET utf8", c.dbName); err != nil {
+	_, err := c.db.Exec("CREATE SCHEMA IF NOT EXISTS " + c.dbName + " DEFAULT CHARACTER SET utf8")
```

Comment:
Was there a reason for moving this from the parameterized style to the string concat?